files/src/hobby_hub_utils/test/test_pin.py

```diff
@@ -0,0 +1,20 @@
+from hobby_hub_pkg.pin_manager import get_led
+import unittest
+import time
+
+class TestPinManager(unittest.TestCase):
+
+    def test_blink_led0(self):
+        led = get_led("myled0")
```

Comment:
what about some of the other methods? I don't think get_gpio would be different but i'd appreciate a test case for at least this as well.

Reply:
Sure, @DeclanBruce  @iliescua

Reply:
@anthony-bartman